### ISSUE TITLE
my world can't open again

### ISSUE THREAD

**Issue by Danyb04** [minecraft-linux/mcpelauncher-manifest#512]
hi,
one day, I was playing on my world with a friend. Suddenly, when I fall into lava, the game crashed. Since that event, I can't open my world again: when I open the world, he loads, then he shows a blue and after black screen, and the application couldn't work. What is happened? is the world corrupted or something?

**Comment by ChristopherHX** [comment]
I suggest copy your world to your android device (or emulator) to recover your world
https://mcpelauncher.readthedocs.io/en/latest/faq.html#where-are-my-worlds
- Open it on android
- move out of lava and water, these are dangerous places of 1.16.210-1.17.0
- Copy it back into the launcher

**Comment by Danyb04** [comment]
I tried to move the file into my android, I use android file transfer and I move the world into the folder minecraftworlds, but he doesn't seem to be any change into the game. What kind of path I need to follow?

**Comment by ChristopherHX** [comment]
You need to copy the folder of your world (from minecraftworlds to minecraftworlds on android). Reopen the game on Android, it should list your world without additional work (make shure you have selected external storage, you cannot access internal storage by other apps). You can also zip your world and share it here, I could then try to fix it myself if you want.

**Comment by Danyb04** [comment]
i solved the issue and open the world.
Idk how to explain, the world doesn't exist. I can get into the world, but I can just see a black screen, that becomes blue. I could also hear the sound of the mobs, but I couldn't do nothing.
I don't have a backup, so at this point I suppose the world is gone out.

**Comment by ChristopherHX** [comment]
Upload your world and share it here, I don't think it is lost forever unless you deleted it already.

**Comment by Danyb04** [comment]
https://drive.google.com/file/d/1Z2G7_4BzCckEQAO6wWaD29bLSl6G8SHI/view?usp=sharing
this is the zipped file, you can download it. Let me know if you can do something, I'm really tied up with this world.

**Comment by ChristopherHX** [comment]
Good news, your world is explorable via local multiplayer (I opened it on my android phone) so your building are recoverable
Bad news, the local player data are completely destroyed and I have no idea how to fix them

I restored some parts of your world, which seems to be working so far
[IyGqYOteFwA=.zip](https://github.com/minecraft-linux/mcpelauncher-manifest/files/6527868/IyGqYOteFwA.zip)

You might want to change back to survival
- Press T
- Type `/gamemode 0`

**Comment by Danyb04** [comment]
thank you so much, you saved 3 months of work:)))))
about player destroyed, what does that mean? could my friend take access into the world?
i also noticed that all the world is ok, but the end dimension party recover (we fought the dragon but he was here again, the stronghold eyes aren't replaces, the nether portal that brings to the stronghold doesn't exist) but all the world is ok.
thank you so much again for you help:)))

**Comment by ChristopherHX** [comment]
> could my friend take access into the world?

Yes, as long the game only show black or blue and keep's open it is possible to join your world from another device and play normally without the missing stuff

> about player data destroyed, what does that mean?
The local player is inivible, cannot move etc.

I deleted a lot of data of your world, so yes there is data loss backup your world frequently to avoid such problems in the future. I'm glad to help.

**Comment by MrSlixX** [comment]
The same thing happens to me but it is with a world of my friend and I can not mess with my account

**Comment by anullihate** [comment]
here's a hack I do to fix the broken player data (the one that feels like your stuck in a void and can't move)

1. **first**, you can move your world to your android phone or if your like me that plays on BDS (Bedrock Dedicated Server) you don't need to move anything.
2. **second**, join to your world in your **android phone**, if you do it in launcher for some reason it will consume too much ram and crash again (for me at least).
3. **third**, if your in android you must have your cheats enabled, that's the reason why I play in BDS because I don't want to disable achievements in my world.
4. **fourth**, you must have another player logged in to your world.
5. **fifth**, use /tp command to tp to the other player, you should be able to move with your GUI displayed but the world is still void or not rendered.
6. **last**, re joined to your world.

for now this is the method I use to fix it without touching the world data. also it's not guaranteed to work in your case until you've tried atleast.

next time, be cautious with water or lava now. if you have something that deals with swimming with water I usually do it with my phone or at least hope that it won't crash.
